datafusion/sqllogictest/test_files/parquet.slt

```diff
@@ -885,3 +885,42 @@ WHERE b = 2;
 
 statement ok
 DROP TABLE t;
+
```

Comment:
It seems only test ASC NULLS FIRST. Better to add test cases for:
     - DESC ordering
     - Multi-column ordering
     - Files with no ordering metadata